x-pack/plugin/analytics/src/main/java/org/elasticsearch/xpack/analytics/boxplot/Boxplot.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.analytics.boxplot;
+
+import org.elasticsearch.search.aggregations.metrics.NumericMetricsAggregation;
+
+public interface Boxplot extends NumericMetricsAggregation.MultiValue {
```

Comment:
Hmm, I'm not sure tbh.  We _might_ want to keep them around as a bridge to the HLRC?  E.g. `InternalMin` and `ParsedMin` both implement `Min`, and it might help us make sure they don't diverge?

Or maybe useless bloat now that the TC is gone and we can get rid of them... but probably better to investigate that in a separate PR and apply it to all aggs if we decide we can remove them?

Reply:
The HLRC won't have `Boxplot` on the classpath because it doesn't have the `analytics` module. With `top_metrics` I just pointed folks to `ParsedTopMetrics`. I'm fine either way. It is just that when I made my new agg I figured I could save myself a little work an not make it.

Reply:
Argh, you're right.  Keep forgetting this is in xpack.  :+1: probably not necessary given the current situation